samples/SkiaAwtSample/src/main/kotlin/SkiaAwtSample/App.kt

```diff
@@ -22,7 +24,15 @@ fun main(args: Array<String>) {
 }
 
 fun createWindow(title: String, exitOnClose: Boolean) = SwingUtilities.invokeLater {
-    val skiaLayer = SkiaLayer()
+    val renderingHints = Toolkit.getDefaultToolkit().getDesktopProperty("awt.font.desktophints") as RenderingHints
```

Comment:
```suggestion
    val renderingHints = Toolkit.getDefaultToolkit().getDesktopProperty("awt.font.desktophints") as Map<Any, Any>
```

Because it fails on macOs:
```
Exception in thread “AWT-EventQueue-0” java.lang.ClassCastException: class java.util.HashMap cannot be cast to class java.awt.RenderingHints (java.util.HashMap is in module java.base of loader ‘bootstrap’; java.awt.RenderingHints is in module java.desktop of loader ‘bootstrap’)
	at SkiaAwtSample.AppKt.createWindow$lambda$1(App.kt:27)
```